github-package-helm-dispatch.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+VERSION_FILE=${VERSION_FILE:="/workspace/version.txt"}
+GITHUB_WORKFLOW=${GITHUB_WORKFLOW:="package-helm.yaml"}
+
+if [ -z "${VERSION}" ]; then
+  VERSION=$(cat "$VERSION_FILE")
+fi
+
+# Final check that version is set
+if [ -z "${VERSION}" ]; then
+  echo "ERROR: VERSION could not be determined"
+  echo "If version file is in different location than ${VERSION_FILE}"
+  echo "before running github_workflow_dispatch.sh script, set a variable:"
+  echo "export VERSION_FILE=/path/version.txt"
+  echo "or set VERSION variable directly'"
+  echo 'export VERSION="$(node|cat ...)"'
+  exit 1
+fi
+
+TAG_TO_USE=${CIRCLE_TAG}  # Circle CI
```

Comment:
No need to have this, since the script is only for Jenkins. (Similarly, in scripts/circle-ci/github-package-helm-dispatch.sh, we do not need code for Jenkins)

Reply:
Will review both versions in a later PR